docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,164 @@
+| Bloom                  | This enables a visual effect known as "Bloom". A slight Bloom effect could be used to make  |
+|                        | zones appear bright.                                                                        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Avatar Priority        | This property changes the update priority of avatar movements by allocating more bandwidth, |
+|                        | which has been reserved by the avatar mixer. It is commonly used on live events to have     |
+|                        | entertainers avatars' movements still be smooth, even with a huge crowd also being updated. |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^^^^^^
+Key Light
+^^^^^^^^^
+
+Key Lighting is used to illuminate a zone. You can think of the Key Light as a sun, which illuminates a zone from a specific direction.
+The following properties are available for Key Lighting:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Key Light Color        | Sets the color of the key light. This could be used to simulate a sunset.                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Light Intensity        | This changes the intensity of the key lighting.                                             |
++------------------------+---------------------------------------------------------------------------------------------+
+| Light Horizontal Angle | Sets the rotation around the zones y axis and starts in its -z direction.                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Light Vertical Angle   | Sets the rotation around the zones x axis and starts in its -z direction.                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Cast Shadows           | This enables shadows, while still respecting the "Cast Shadows" property of other entities. |
++------------------------+---------------------------------------------------------------------------------------------+
+| Shadow Bias            | This controls shadow bias and needs to be fine tuned. Too low values can cause so called    |
+|                        | "shadow acne", while too high values can cause so called "peter panning".                   |
```

Comment:
```suggestion
|                        | "shadow acne", while too-high values can cause so called "peter panning".                   |
```

Reply:
Changing `Too high` to `Too-high` seems pretty weird to me.

Reply:
OK